assets/third_party/sounds/player/Foot.ogg.license

```diff
@@ -0,0 +1,4 @@
+﻿SPDX-FileCopyrightText: d00121058
+SPDX-License-Identifier: CC0-1.0
+
+FX_001_footstep_grass_l.wav by d00121058 -- https://freesound.org/people/d00121058/sounds/390760/
```

Comment:
Are you sure this is the right sound? To me the sound in the PR sounds very different to this linked sound.

Reply:
I may have mixed it up with the one in the stealth challenge!, let me check

Reply:
It seems the license was wrong in the original PR too D:, I'll check with @Arebuayon , if I can't find the original license I'll get a new sound